test/commands/environment/set-variables.test.js

```diff
@@ -255,6 +255,9 @@ test('set-environment-variables - delete secret', async () => {
 })
 
 test('set-environment-variables - delete not found', async () => {
+  const originalWarn = console.warn
```

Comment:
This change does not seem related to the PR. I'm not sure why this is necessary, but please create a separate issue/PR for this.

Reply:
@justinedelson I think the tests are broken honestly without this. I will create a separate pr.